osi_object.proto

```diff
@@ -1062,18 +1073,191 @@ message MovingObject
+        // the pedestrian model used. For example, some simulators will not include
+        // detailed data about the hands of a pedestrian.
+        //
+        repeated Bone skeleton_data = 2;
+
+        //
+        // \brief Bones in the skeleton of the pedestrian.
+        //
+        // Each point represents a joint, or otherwise important point in the skeleton
+        // of a pedestrian. For example pelvis, knee or shoulder. The naming convention
+        // should be followed for identifying bones.
+        //
+        message Bone
+        {
+            // Bones are identified by their type, combined with which body side
+            // they are on.
+            //
+            // To properly identify the bones the pre-defined naming convention
+            // must be used.
+            //
+            // \image html OSI_SkeletonNamingConvention.svg
+            //
+            // A bone's name, position, and orientation is defined by the end-point closer
+            // to the root. For example: the "LOWER_ARM_L" will define the point in the left
+            // elbow of the model.
+            //
+            // If a bone which is more than one layer detached from the
+            // root point is used, all bones between that bone and the root also
+            // need to be defined in order to create a complete chain!
+            //
+            // If information about bones are missing, they may be left empty.
+            //
+            optional Type type = 1;
+
+            // Position of the bone.
+            //
+            // Reference System is the root, defined by bbcenter_to_root
+            // (\c PedestrianAttributes::bbcenter_to_root).
+            //
+            optional Vector3d position = 2;
+
+            // Orientation of the bone.
+            //
+            // Reference System is the root, defined by bbcenter_to_root
+            // (\c PedestrianAttributes::bbcenter_to_root).
+            //
+            optional Orientation3d orientation = 3;
+
+            // Orientation of the bone.
+            //
+            // Reference System is the root, defined by bbcenter_to_root
+            // (\c PedestrianAttributes::bbcenter_to_root).
+            //
+            optional float length = 4;
+
+            // Determines a bone is explicitly missing from the pedestrian model.
+            //
+            // In case a pedestrian has missing limbs this can be explicitly communicated
+            // by setting this boolean to TRUE.
+            //
+            optional bool missing = 5;
+
+            // The type of the bone.
+            //
+            // \note Bone of each type can be assigned, or left empty depending
+            // on the desired level of detail, or present data. However, if a
+            // bone is defined, all bones in the chain from that bone back to
+            // the root point must be defined to create a complete chain.
+            //
```

Comment:
I suggest moving this comment to the field definition, where level of detail is already mentioned. This is more a use thing than an enum type thing.
```suggestion
```